package-lock.json

```diff
@@ -9716,5 +9716,7844 @@
                 "url": "https://github.com/sponsors/sindresorhus"
             }
         }
+    },
```

Comment:
Have you checked if these dependency changes are fine? If you havent or are not sure about them just reset them